omegarecyclerviewlibs/src/main/java/com/omega_r/libs/omegarecyclerview/expandable_recycler_view/animation/ExpandableItemAnimator.java

```diff
@@ -0,0 +1,731 @@
+        Runnable mover = new Runnable() {
+            @Override
+            public void run() {
+                for (MoveInfo moveInfo : moves) {
+                    animateMoveImpl(moveInfo.holder, moveInfo.fromX, moveInfo.fromY, moveInfo.toX, moveInfo.toY);
+                }
+                moves.clear();
+                mMovesList.remove(moves);
+            }
+        };
+        if (hasRemovals) {
+            View view = moves.get(0).holder.itemView;
+            ViewCompat.postOnAnimationDelayed(view, mover, getRemoveDuration());
+        } else {
+            mover.run();
+        }
+    }
+
+    private void runChangesActions(boolean hasRemovals) {
+        final ArrayList<ChangeInfo> changes = new ArrayList<>(mPendingChanges);
+        mChangesList.add(changes);
+        mPendingChanges.clear();
+        Runnable changer = new Runnable() {
+            @Override
+            public void run() {
+                for (ChangeInfo change : changes) {
+                    animateChangeImpl(change);
+                }
+                changes.clear();
+                mChangesList.remove(changes);
+            }
+        };
+        if (hasRemovals) {
+            ViewHolder holder = changes.get(0).oldHolder;
+            ViewCompat.postOnAnimationDelayed(holder.itemView, changer, getRemoveDuration());
+        } else {
+            changer.run();
+        }
+    }
+
+    private void runAddActions(boolean hasRemovals, boolean hasMoves, boolean hasChanges) {
+        final ArrayList<ViewHolder> additions = new ArrayList<>(mPendingAdditions);
+        mAdditionsList.add(additions);
+        mPendingAdditions.clear();
+        Runnable adder = new Runnable() {
+            public void run() {
+                proceedWithAnimationHelper(additions, new UnVoidFunction<ViewHolder>() {
+                    @Override
+                    public void apply(ViewHolder param) {
+                        animateAddImpl(param);
+                    }
+                }, true);
+                mAdditionsList.remove(additions);
+            }
+        };
+        if (hasRemovals || hasMoves || hasChanges) {
+            long removeDuration = hasRemovals ? getRemoveDuration() : 0;
+            long moveDuration = hasMoves ? getMoveDuration() : 0;
+            long changeDuration = hasChanges ? getChangeDuration() : 0;
+            long totalDelay = removeDuration + Math.max(moveDuration, changeDuration);
+            View view = additions.get(0).itemView;
+            ViewCompat.postOnAnimationDelayed(view, adder, totalDelay);
+        } else {
+            adder.run();
+        }
+    }
+
+    private void proceedWithAnimationHelper(List<ViewHolder> holders, UnVoidFunction<ViewHolder> func, boolean isAdding) {
+        int childChangesCount = 0;
+        for (int i = 0; i < holders.size(); i++) {
+            ViewHolder holder = holders.get(i);
+            if (holder instanceof OmegaExpandableRecyclerView.Adapter.ChildViewHolder) {
```

Comment:
OmegaExpandableRecyclerView.ChildViewHolder

Reply:
Not possible